src/sentry/api/helpers/source_map_helper.py

```diff
@@ -23,7 +29,25 @@
 }
 
 
-def source_map_debug(project, event_id, exception_idx, frame_idx):
+class SourceMapDebug:
+    def __init__(self, issue: str | None = None, data: Mapping[str, Any] | None = None) -> None:
+        self.issue = issue
+        self.data = data
+
+
+class SourceMapException(Exception):
+    def __init__(self, issue: str, data: Mapping[str, Any] | None = None) -> None:
+        super().__init__(issue, data)
+        self.issue = issue
+        self.data = data
```

Comment:
these won't need reordering if you add `from __future__ import annotations` to this file

that said, I kinda like to put little helpers and exceptions at the top anyway so 👍